assets/src/application/ui/forms/ee-form/FieldRenderer.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { FormItem, MappedField } from './adapters';
```

Comment:
Maybe just `Field`, not `MappedField`

Reply:
because it's clear that it's from `'./adapters'`, so it's not a lib-based field

Reply:
May be I wanted to be more expressive 😄